vector/src/main/java/im/vector/adapters/RoomViewHolder.java

```diff
@@ -330,10 +333,12 @@ public void onClick(View v) {
                     vRoomNotificationMute.setVisibility(View.GONE);
                     break;
                 case MENTIONS_ONLY:
-                    if (room.isDirect()) {
-                        // Tchap: This mode is not suggested anymore for the direct chats
-                        // We consider people will not mention the other member in 1:1.
+                    if ((pushManager.useFcm() && room.isEncrypted()) || room.isDirect()) {
```

Comment:
If a room is not encrypted we should have mentions only option ?

Reply:
yes we support the mentions only in this case